lighthouse-cli/test/smokehouse/run-smoke.js

```diff
@@ -63,7 +63,7 @@ const SMOKETESTS = [{
   id: 'offline',
   expectations: 'offline-local/offline-expectations.js',
   config: smokehouseDir + 'offline-config.js',
-  batch: 'parallel-second',
+  batch: 'offline',
```

Comment:
@paulirish I made this change after too many flakes on travis, now that things aren't throttled it seems too much is trying to happen at once and at least one navstart was getting dropped on ~70% of runs :(

Reply:
aiight.